src/libs/JmWalletApi.js

```diff
@@ -155,6 +155,23 @@ const postFreeze = async ({ walletName, token }, { utxo, freeze = true }) => {
   })
 }
 
+/**
+ * Get the value of a specific config setting. Note values are always returned as string.
+ *
+ * @returns an object with property `configvalue` as string
+ */
+const postConfigGet = async ({ walletName, token, signal }, { section, field }) => {
```

Comment:
Why did they make this a POST call. 😅 😭 Really stretches our naming scheme...

Should we start making exceptions and just go for `getConfig` or so?

Reply:
As you please. I do not have a strong opinion until we generate the client from the openapi spec : D
But if we deviate from the current naming scheme, we should reconsider all other method names too.

Should we just ditch the http method prefix and reorder?
e.g. `postMakerStart` -> `startMaker`
`postDirectSend` -> `sendDirect`
?

I'd do that in a separate PR if that is okay with you.
(Let's not touch files that have nothing to do with the task at hand)

Reply:
Let's forget it for now and do a refactor of the API stuff another time. Then we can also improve other things like drying up response parsing etc.